pages/api/verification/complete.ts

```diff
@@ -169,6 +170,24 @@ export default async function handler(req: NextApiRequestWithBody, res: NextApiR
     }
 
     const assignedRoles = guild.roles.filter((role) => roleIds.includes(role.id))
+    const posthog = PostHogClient()
+
+    const captureResult = await posthog.capture({
+      event: 'Discord Integration verification',
```

Comment:
```suggestion
      event: 'discord integration verification',
```
for consistency with other apps